rosidl_generator_py/package.xml

```diff
@@ -29,6 +29,8 @@
   <buildtool_export_depend>rosidl_typesupport_c</buildtool_export_depend>
   <buildtool_export_depend>rosidl_typesupport_interface</buildtool_export_depend>
 
+  <exec_depend>ament_cmake_flake8</exec_depend>
+  <exec_depend>ament_cmake_pep257</exec_depend>
```

Comment:
Ah interesting. It seems like we would need a `test_export_depend` option to exist to describe these since they're needed only for tests in downstream packages. Adding them as `exec_depend` means that they'll always be installed when a message package is installed, but these dependencies are probably light enough that this is fine.

Reply:
> Adding them as `exec_depend` means that they'll always be installed when a message package is installed, but these dependencies are probably light enough that this is fine.

Ah, that makes sense (assuming you meant excluding test deps to be done by [`rosdep install -t`](https://github.com/ros-infrastructure/rosdep/pull/789)). Although I'm unsure how many users carefully exclude test deps, upstream pkg like this one should do the right thing. I'll update the PR.

> `test_export_depend`

:sweat_smile:

Reply:
Updated, please re-review.